ct/suwayomiserver.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: tremor021
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://github.com/Suwayomi/Suwayomi-Server
+
+APP="Suwayomi-Server"
```

Comment:
Should be APP="SuwayomiServer" or the installation script name must be install/suwayomi-server-install.sh